creator-node/src/services/stateMachineManager/index.js

```diff
@@ -49,54 +53,71 @@ class StateMachineManager {
       prometheusRegistry
     )
 
-    // Upon completion, make queue jobs record metrics and enqueue other jobs as necessary
-    const queueNameToQueueMap = {
-      [QUEUE_NAMES.MONITOR_STATE]: {
-        queue: monitorStateQueue,
-        maxWaitingJobs: 10
-      },
-      [QUEUE_NAMES.FIND_SYNC_REQUESTS]: {
-        queue: findSyncRequestsQueue,
-        maxWaitingJobs: 10
-      },
-      [QUEUE_NAMES.FIND_REPLICA_SET_UPDATES]: {
-        queue: findReplicaSetUpdatesQueue,
-        maxWaitingJobs: 10
-      },
-      [QUEUE_NAMES.MANUAL_SYNC]: {
-        queue: manualSyncQueue,
-        maxWaitingJobs: 1000
-      },
-      [QUEUE_NAMES.RECURRING_SYNC]: {
-        queue: recurringSyncQueue,
-        maxWaitingJobs: 100000
-      },
-      [QUEUE_NAMES.UPDATE_REPLICA_SET]: {
-        queue: updateReplicaSetQueue,
-        maxWaitingJobs: 10000
-      },
-      [QUEUE_NAMES.RECOVER_ORPHANED_DATA]: {
-        queue: recoverOrphanedDataQueue,
-        maxWaitingJobs: 10
-      }
+    if (cluster.worker?.id === 1) {
+      await SyncRequestDeDuplicator.clear()
     }
-    for (const [queueName, { queue }] of Object.entries(queueNameToQueueMap)) {
-      queue.on(
-        'global:completed',
-        makeOnCompleteCallback(
-          queueName,
-          queueNameToQueueMap,
-          prometheusRegistry
-        ).bind(this)
-      )
+    if (clusterUtils.isThisProcessSpecial()) {
```

Comment:
i don't follow why you check if processisSpecial here, but on line 56 you check if it === 1
this is a duplicated comment from index.ts - lets discuss all at once